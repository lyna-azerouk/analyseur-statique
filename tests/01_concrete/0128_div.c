{
  int x;
  x = rand(100,105) / rand(2,4);
  print(x);
}
