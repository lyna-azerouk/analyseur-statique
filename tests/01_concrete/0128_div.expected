tests/01_concrete/0128_div.c:4.2-11: { [x=25;]; [x=26;]; [x=33;]; [x=34;]; [x=35;]; [x=50;]; [x=51;]; [x=52;]; }
analysis ended
